service_info_cms/admin.py

```diff
@@ -1,10 +1,25 @@
 from django.contrib import admin
 from cms.extensions import PageExtensionAdmin
 
-from .models import IconNameExtension
+from .models import IconNameExtension, RatingExtension, PageRating
 
 
 class IconNameExtensionAdmin(PageExtensionAdmin):
     pass
 
+
+class RatingExtensionAdmin(PageExtensionAdmin):
+    pass
+
+
+class PageRatingAdmin(admin.ModelAdmin):
+    list_display = ('page_obj', 'num_ratings', 'average_rating',)
+    readonly_fields = ('page_obj', 'num_ratings', 'average_rating', )
```

Comment:
add rating_total to readonly_fields too (doesn't seem to bring happiness if you change it in admin, since it doesn't affect the average immediately)